src/queue.js

```diff
@@ -216,6 +220,7 @@ const queue = {
   },
 
   remove: (username) => {
+    twitch.notLurkingAnymore(username); // If a user is leaving, they're not lurking
```

Comment:
Unfortunately the argument `username` here is the display name of someone. This function would need to be changed to also accept the username (or only the username), such that `twitch.notLurkingAnymore` can be called with the username, and not display name.

Reply:
Resolved this by moving it out of queue.js and into index.js, where we have the username accessible.